llvm/test/Transforms/LoopVectorize/RISCV/vectorize-force-tail-with-evl-reduction.ll

```diff
@@ -0,0 +1,1495 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -passes=loop-vectorize \
```

Comment:
this only tests in loop reductions, should it be indicated in the test  file name?

Reply:
As I mentioned, if changing the title is possible, we can directly add out-loop reduction tests to this test case.

Reply:
Sure, but the checks for regular reductions are still missing? As the ordered case has been split off, it would be consistent to also split off in loop tests to a separate file, to avoid a single file becoming massive

Reply:
3b527754bfb3a3e27cd729d15f511608e2f7f367
Separated outloop and inloop reduction into two files.